### PR TITLE
#19 Added imap and its build dependency

### PR DESCRIPTION
**Pull request by krufab**
So that the the warning from setup.php disappears

Fixes: #19

### CODE REVIEW

File: Dockerfile-alpine.template [hunk 1]
```diff
@@ -9,10 +9,12 @@ RUN apk add --no-cache \
 RUN set -ex; \
 	\
 	apk add --no-cache --virtual .build-deps \
+    imap-dev \
```
Comment (J0WI) [inline Dockerfile-alpine.template:12]: Please use tabs to indent lines.

Comment (krufab) [reply Dockerfile-alpine.template:12]: Changed

**Comment by J0WI** [comment]
Now the Debian variant has kerberos support while the Alpine variant has not. [Here](https://github.com/nextcloud/docker/tree/master/.examples/dockerfiles/imap) you can find an example for proper imap support.

File: Dockerfile-alpine.template [hunk 2]
```diff
@@ -9,10 +9,17 @@ RUN apk add --no-cache \
 RUN set -ex; \
 	\
 	apk add --no-cache --virtual .build-deps \
-		sqlite-dev \
+		imap-dev \
+		krb5-dev \
+		libressl-dev \
```
Comment (J0WI) [inline Dockerfile-alpine.template:14]:
```suggestion
		openssl-dev \
```

Comment (J0WI) [reply Dockerfile-alpine.template]: Please run `update.sh` again

**Review by J0WI — APPROVED** [review]

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**